scheduler/src/cook/mesos/task.clj

```diff
@@ -1,50 +1,57 @@
 (ns cook.mesos.task
   (:require [cook.mesos.util :as util]
             [plumbing.core :refer (map-vals)]
-            [clojure.tools.logging :as log])
+            [clojure.tools.logging :as log]
+            [cheshire.core :as json])
   (import com.netflix.fenzo.TaskAssignmentResult))
 
 (defonce custom-executor-name "cook_agent_executor")
 (defonce custom-executor-source "cook_scheduler")
 
+(defn build-task-status-data [job-ent]
+  (.getBytes
+   (if (:job/custom-executor job-ent true)
+     ;;TODO this data is a race-condition
+     (pr-str
+      {:instance (str (count (:job/instance job-ent)))})
+     (json/generate-string
```

Comment:
To maintain backwards compatibility with existing clojure executor that expects a serialized clojure data structure.